provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/ProvisioningDeviceClientRegistrationResult.java

```diff
@@ -7,12 +7,33 @@
 
 package com.microsoft.azure.sdk.iot.provisioning.device;
 
+import lombok.Getter;
+
 public class ProvisioningDeviceClientRegistrationResult
```

Comment:
This is the public facing class that our provisioning device client returns to a user after a successful device registration. Previously, it only had the essential fields exposed, but now it exposes all the fields that the service gives us.